src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -90,6 +96,7 @@ public async Task<IEnumerable<Type>> GetExtensionsStartupTypesAsync()
             }
             else
             {
+                _logger.ScriptStartNotLoadingExtensionBundle(_rootScriptPath, bundleConfigured, isPrecompiledFunctionApp, isLegacyExtensionBundle);
                 binPath = Path.Combine(_rootScriptPath, "bin");
```

Comment:
is this going to log the right thing? Should we pass `binPath`, rather than `_rootScriptPath` into the log statement?

Reply:
was kind of debating logging the binPath, tried to keep it consistent with the log statement for bundles. Also for some reason if Path.Combine throws an exception we won't lose the log. Don't think path combine would throw though

Reply:
Fixed in follow up commit